src/System.DirectoryServices.AccountManagement/tests/PrincipalContextTests.cs

```diff
@@ -90,7 +93,8 @@ public void Ctor_ContextType_Name_Container_Options(ContextType contextType, str
             }
             else
             {
-                Assert.Throws<InvalidCastException>(() => context.ConnectedServer);
+                Assert.NotNull(context.ConnectedServer);
```

Comment:
Ah, just to be sure: do you mean that `Assert.Equal((string)null, (string)null)` will assert anyway? I know that it is unlikely to be a problem to have just the `Assert.Equal`, since it requires `Environment.MachineName` to be also broken, but given that `null` has a specific meaning for [`ConnectedServer`](https://msdn.microsoft.com/en-us/library/system.directoryservices.accountmanagement.principalcontext.connectedserver(v=vs.110).aspx) property I want to confirm that.